src/main/java/io/aiven/kafka/connect/common/output/jsonwriter/JsonLinesOutputWriter.java

```diff
@@ -26,22 +26,26 @@
 
 public class JsonLinesOutputWriter extends OutputWriter {
 
+    public JsonLinesOutputWriter(final Collection<OutputField> fields, final OutputStream outputStream,
+                                 final boolean jsonEnvelopeEnabled) {
+        super(outputStream, new Builder().addFields(fields).build(jsonEnvelopeEnabled));
+    }
+
     public JsonLinesOutputWriter(final Collection<OutputField> fields, final OutputStream outputStream) {
-        super(outputStream, new Builder().addFields(fields).build());
+        super(outputStream, new Builder().addFields(fields).build(true));
     }
 
     static final class Builder {
```

Comment:
It seems this `Builder` should reside in `JsonLinesOutputStreamWriter` as it builds objects of this class. No need to change now, just a comment for the future. (Same in `JsonOutputWriter`.)